src/notification-parser.ts

```diff
@@ -189,7 +193,13 @@ export function parseNotificationOpts<T extends OptionValues>(
     notifications.push({ type: NotificationType.NONE })
   }
 
-  logger.debug('Configured notifications: ' + notifications.map(n => n.type))
+  logger.debug(
+    'Configured notifications: ' +
+      notifications
+        .map(n => n.type)
+        .map(n => `\n- ${n}`)
```

Comment:
```suggestion
        .map(n => `\n- ${n.type}`)
```
(non-blocking)
Not important, but this could be simplified to a single `map`.

Reply:
agree. I was somehow thinking of splitting it, but when looking at it for the second time here, I believe it is nonsense not to compose it into a single map.
I do not do "commit suggestion" as I'm afraid that would require a 3rd person to acknowledge the PR.